src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/publiseringsdatoer/api/PubliseringsdatoerController.java

```diff
@@ -0,0 +1,36 @@
+package no.nav.arbeidsgiver.sykefravarsstatistikk.api.publiseringsdatoer.api;
+
+import javax.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.publiseringsdatoer.DatauthentingFeil;
+import no.nav.security.token.support.core.api.Unprotected;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@Slf4j
+@Unprotected
```

Comment:
Er det nødvendig å ha `Unprotected`? 
Det er kilde for angrep om vi gjør det.

Reply:
tanken var at api-et skal være åpent, derfor satte vi på den, så man ikke trenger å autentisere seg for å bruke endepunktet. burde vi endre?